Cargo.toml

```diff
@@ -30,6 +29,7 @@ semver = "0.2.1"
 toml = "0.1"
 unicode-normalization = "0.1"
 quine-mc_cluskey = "0.2.2"
+clippy_lints = { version = "0.0.*", path = "clippy_lints" }
```

Comment:
Actually, it occurred to me that we might want to hardcode the version so as to avoid upgrade pains -- people shouldn't have to `cargo update -p clippy` and then still get errors because they didn't upgrade clippy_lints. Thoughts?